### PR TITLE
Adding Vanilla Tweaks Options

### PR DESCRIPTION
**Pull request by shotah**
Thanks for your time reviewing these!

### CODE REVIEW

**Comment by shotah** [comment]
Linked Issue: https://github.com/itzg/docker-minecraft-server/issues/1240

**Comment by shotah** [comment]
@itzg Added setup only and sharecode feature.

File: examples/vanilla-tweaks/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      EULA: "TRUE"
+      VERSION: ${MINECRAFT_VERSION:-LATEST}
+      VANILLATWEAKS_FILE: /config/vanillatweaks-datapacks.json
+      REMOVE_OLD_VANILLATWEAKS: "TRUE"
+    volumes:
+      - data:/data
+      - ./vanillatweaks-datapacks.json:/config/vanillatweaks-datapacks.json:ro
+  vanillatweaks_sharecode:
+    ports:
+      - "25565:25565/tcp"
```
[the remark this answers is not in the document]
Comment (shotah) [reply examples/vanilla-tweaks/docker-compose.yml]: done!

File: tests/test.sh [hunk 2]
```diff
@@ -1,22 +1,56 @@
...
 down() {
-  docker-compose down -v
+  docker-compose down -v --remove-orphans
 }
 
-docker-compose run monitor || failed=true
-echo "
-Result: failed=$failed"
+fullMinecraftUpTest(){
+  folder=$1
```
[the remark this answers is not in the document]
Comment (shotah) [reply tests/test.sh]: done!

**Review by itzg — APPROVED** [review]
Thanks for the quick fixes and diligence.

**Comment by shotah** [comment]
> Thanks for the quick fixes and diligence.

Hey thanks for really owning this package! I've used it, and want to commit back. It's saying something that I haven't found a reason to jump in until I noticed the gap with the VT guys. They were kind enough to walk me through their API's and were extremely interested to see this get implemented.

**Comment by itzg** [comment]
Image build is going https://github.com/itzg/docker-minecraft-server/actions/runs/1652062177

**Pull request merged**